src/IconButtonCatalog.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import ComponentCatalogPanel from './ComponentCatalogPanel.js';
+import {MDCIconButtonToggle} from '@material/icon-button';
+import {MDCRipple} from '@material/ripple';
+
+import './styles/IconButtonCatalog.scss';
+
+const IconToggleCatalog = () => {
+  const description = 'Icons are appropriate for buttons that allow a user to take actions or ' +
+                      'make a selection, such as adding or removing a star to an item. ';
+  return (
+    <ComponentCatalogPanel
+      hero={<IconButtonToggle/>}
+      title='Icon Button'
+      description={description}
+      designLink='https://material.io/go/design-buttons'
```

Comment:
https://material.io/design/components/buttons.html#toggle-button to match main repo's link?